pom.xml

```diff
@@ -204,6 +204,14 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
   </dependencies>
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.7</source>
+          <target>1.7</target>
+        </configuration>
+      </plugin>
```

Comment:
@paulodamaso Why did you add this? :D

Reply:
@amihaiemil My IDE was complaining about it, but it was some local coniguration.